python/test/unit/hopper/test_persistent_warp_specialized_fused-attention.py

```diff
@@ -321,7 +326,11 @@ def test_op(Z, H, N_CTX, D_HEAD, dtype=torch.float16):
     tri_dq, q.grad = q.grad.clone(), None
     # compare
     assert torch.allclose(ref_out, tri_out, atol=1e-2, rtol=0)
-    assert torch.allclose(ref_dv, tri_dv, atol=1e-2, rtol=0)
+    # Bigger tolerance for AMD MI200 devices.
```

Comment:
is this getting fixed or discussed somewhere. It seems like it going to be a problem long term and this is just a workaround

Reply:
It's basically a hardware limit--the link below explains more. Yeah we chatted about this issue. Concensus is that we don't care about MI200 that much going forward.

Reply:
got it, thanks for explaining. Do we plan to drop testing for MI200 at some point?

Reply:
Yeah need to wait for mi300 bots to be added first and then we can consider drop mi200 ones.